mods/ts/rules/nod-support.yaml

```diff
@@ -270,6 +271,19 @@ NASAM:
 NASTLH:
 	Inherits: ^Building
 	Inherits@IDISABLED: ^DisabledOverlay
+	HitShape:
+		UseOccupiedCellsOffsets: false
+		TargetableOffsets: 512,128,0, 1024,-512,0, -256,1024,0
+		Type: Rectangle
+			TopLeft: 0, -1536
+			BottomRight: 1024, 1536
+	HitShape@lowerSphere:
```

Comment:
I need to find out where/how are these additional 'Hitshape@name' used, style nit: LowerSphere?